<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PGPromotion</class>
 <widget class="QWidget" name="PGPromotion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="PBSave">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PBImport">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineVersion">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>选择一个版本</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PBConfirm">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>140</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PBRefresh">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>100</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>108</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>当前折扣</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>促销</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line1">
        <property name="minimumSize">
         <size>
          <width>31</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>23</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>等级</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line2">
        <property name="minimumSize">
         <size>
          <width>31</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>23</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>企业特供</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line3">
        <property name="minimumSize">
         <size>
          <width>31</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>23</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>积分</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line4">
        <property name="minimumSize">
         <size>
          <width>31</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>23</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>411</x>
     <y>31</y>
     <width>108</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>更改折扣</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>促销</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="linePro">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>31</width>
          <height>23</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>等级</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineLevel">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>31</width>
          <height>23</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>企业特供</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineCor">
        <property name="minimumSize">
         <size>
          <width>31</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>23</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>积分</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="linePoints">
        <property name="minimumSize">
         <size>
          <width>31</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>23</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-40</y>
     <width>191</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:700;&quot;&gt;促销折扣管理&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
